freshCraTemplateUpdate.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
+
+cd ${HOME}/tmp/my-test-app
+# The order of these sed commands is kind of important. The last one of uncommenting all lines needs to occur after
+# removing comments explaining in english steps for replacing the binding path
+sed -i.bak 's/{{BINDING_PATH}}/fresh-cra-template.frontier/' blueprint.yml
```

Comment:
I'm thinking @gmdayley might like to link to this as an example?
I wonder if we should make the path something like:
/frontier/fresh
/frontier/cra
/frontier/example
/frontier/sample

We can also decide later and change it easily